CODE_GENERATION.md

```diff
@@ -157,11 +157,14 @@ in generation and then inject the value of our annotation in its place in the SD
 
 Likely, we need to do this injection before every SDK call. The following is an
 example for hook functions injected:
+// TODO: we mention to create setup.go in the previous step but all existing examples 
```

Comment:
`hooks.go` was generated and then edited by the user but now we don't generate it at all. In the future, we'll likely decrease the amount of code goes in that file but `SetupCRDNAME` function will stay, so I wanted to rename it as `setup.go` to be future-proof.